docs/projects.mdx

```diff
@@ -15,6 +15,7 @@
   Spectacle
 - [Charge][]: An opinionated, zero-config static site generator
 
```

Comment:
Below charge you can add the following to use the link reference you've added below:

```mdx
- [MDNEXT][]: INSERT DESCRIPTION HERE
```